server/root/etc/cont-init.d/30-data

```diff
@@ -1,2 +1,39 @@
-#!/bin/sh
+#!/usr/bin/with-contenv bash
+
+set -euo pipefail
+
+# Cleanup data
+rm -fr /tmp/data ~/.synapseConfig
+
+if [ ! -d "/tmp/data" ]; then
+    # Create ~/.synapseConfig
+    printf "%s\n" \
+        "[authentication]" \
+        "username = ${SYNAPSE_USERNAME}" \
+        "apikey = ${SYNAPSE_API_KEY}" | tee -a ~/.synapseConfig >/dev/null
+
+    mkdir -p /tmp/data
+
+    echo "Download clinical notes from Synapse"
+    synapse get ${DATA_SYNAPSE_ID} --downloadLocation /tmp/data \
+        >/dev/null
+    echo ${DATA_MD5} \
+        /tmp/data/2014-i2b2-nlp-evaluation-data-txt.zip | md5sum --check - \
```

Comment:
@thomasyu888 Is there a way to either get the filename programmatically from `synapse get` or specify to `synapse get` the target filename (i.e. not only the target directory).